snuba/query/query.py

```diff
@@ -61,6 +94,35 @@ def __init__(self, body: MutableMapping[str, Any], data_source: RelationalSource
         self.__final = False
         self.__data_source = data_source
 
+        # New data model
+        # TODO: Provide a better typing for this.
+        self.__selected_columns: Sequence[Expression] = selected_columns or []
+        self.__aggregations: Sequence[NodeAggregation] = aggregations or []
+        self.__array_join = array_join
+        self.__condition: Optional[Expression] = condition
+        self.__groupby: Sequence[Expression] = groupby or []
+        self.__having: Optional[Expression] = having
+        self.__order_by: Sequence[OrderBy] = order_by or []
```

Comment:
I think it can. I suspect I forgot them in place because in the first commit I did not pass the parameters to the constructor but I wanted the attributes to be created. Let me try to remove them.